debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ConnectorConfiguration.java

```diff
@@ -68,14 +66,13 @@ public static ConnectorConfiguration forJdbcContainer(JdbcDatabaseContainer<?> j
         return configuration;
     }
 
-    public static ConnectorConfiguration forMongoDbContainer(MongoDBContainer mongoDbContainer) {
+    public static ConnectorConfiguration forMongoDbContainer(MongoDbContainer mongoDbContainer) {
         final List<Integer> exposedPorts = mongoDbContainer.getExposedPorts();
 
         ConnectorConfiguration configuration = new ConnectorConfiguration();
         configuration.with(CONNECTOR, "io.debezium.connector.mongodb.MongoDbConnector")
-                .with(HOSTS, "rs0/" + mongoDbContainer.getContainerInfo().getConfig().getHostName()
-                        + ":" + exposedPorts.get(0))
-                .with(AUTO_DISCOVER_MEMBERS, false);
+                .with(CONNECTION_STRING, "mongodb://" + mongoDbContainer.getContainerInfo().getConfig().getHostName()
+                        + ":" + exposedPorts.get(0));
```

Comment:
Does this return the port inside the container or the outside port? We should make sure we use the right port here. That might have changed as before inside and outside port were equal and not automatically mapped.

Reply:
I believe the inside port is returned here.

Reply:
I see this also switches to our contaienr. Apology for not noticing

Reply:
@ani-sha  also this is not the correct way to get the address here.

If you need an address which is addressable from client use `MongoDbContainer#getClientAddress()`. If ( as seems to be case here) you only need an address which is reachable within docker network, use `MongoDbContaienr#getNamedAddress()`